### PR TITLE
update for more images in webp

### PR DESCRIPTION
**Pull request by snHolst**
- fixed blog images ($sArticle.media.thumbnails to $sArticle.preview.thumbnails)
- added ajax_add_article.tpl
- added checkout/ajax_add_article.tpl
- added compare/col.tpl
- added frontend/detail/config_variant.tpl
- added frontend/listing/product-box/box-big-image.tpl
- added frontend/listing/product-box/box-eomtion.tpl

### CODE REVIEW

File: Resources/views/frontend/checkout/ajax_add_article.tpl [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{extends file="parent:frontend/checkout/ajax_add_article.tpl"}
+
+            {* Article image *}
+            {block name='checkout_ajax_add_information_image'}
```
Comment (shyim) [inline Resources/views/frontend/checkout/ajax_add_article.tpl]: Can you fix the indention?

File: Resources/views/frontend/compare/col.tpl [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{extends file="parent:frontend/compare/col.tpl"}
+
+                                {block name="frontend_compare_article_picture"}
```
Comment (shyim) [inline Resources/views/frontend/compare/col.tpl]: Here also

**Comment by snHolst** [comment]
hey shyim, i fixed the indention.

**Comment by shyim** [comment]
Thanks :blue_heart:

**Pull request merged**